README.md

```diff
@@ -1,2 +1,24 @@
 # DidIAsk-rs
 Did I ask, rewritten in Rust using serenity-rs
+
+## Getting Started:
+1. Clone this repository
+1. Install Rust and Cargo. [Don't Know How?](https://doc.rust-lang.org/cargo/getting-started/installation.html)
+1. Sign into Discord and get the bot registered. Note down the bot token and take care of it. Also set the permissions to send messages and read message history. [Don't Know How?](https://www.writebots.com/discord-bot-token/)
+1. Using Discord, find out the channel ID you want the bot to run on. [Don't know how?](https://www.swipetips.com/how-to-get-channel-id-in-discord/)
+1. Edit the file `src/main.rs` and replace the array `CHANNEL_LIST` with your channel IDs. Take care to set the size of the array to the correct value as well. Example
```

Comment:
Clarify that these are these are the channels the bot are allowed to randomly "did i ask?"